### PR TITLE
docs: gitlab guide updates

### PR DESCRIPTION
**Pull request by elylucas**

### CODE REVIEW

File: docs/guides/cloud/gitlab-integration.mdx [hunk 1]
```diff
@@ -75,6 +102,22 @@ from merging any MRs with flaky tests.
 
 You can manage this behavior on your project's **Project Settings** page.
 
+<DocsImage
+  src="/img/dashboard/gitlab-integration/gitlab-status-checks.jpg"
+  width={800}
+/>
+
+:::tip
```
Comment (elylucas) [inline docs/guides/cloud/gitlab-integration.mdx:110]: @pstakoun let me know what you think of these Flakey test CTAs and if you want any changes to them.

File: docs/guides/cloud/github-integration.mdx [hunk 2]
```diff
@@ -375,6 +375,17 @@ integration settings:
   alt="Status checks settings"
 />
 
+:::tip
+
+<strong>
```
Comment (elylucas) [inline docs/guides/cloud/github-integration.mdx:380]: I missed this in the github guide updates so adding it here. @pstakoun like the gitlab one, let me know if you want any changes to this CTA.

File: docs/guides/cloud/github-integration.mdx [hunk 3]
```diff
@@ -375,6 +375,17 @@ integration settings:
...
+  <Icon name="star" /> Premium Cypress Cloud Feature
+</strong>
+
+Reporting on flakey status checks is included in our
+[Team, Business and Enterprise paid pricing plans](https://www.cypress.io/pricing).
+
+:::
+
 ## Pull request comments
```
Comment (pstakoun) [inline docs/guides/cloud/github-integration.mdx:389]: I realize that the bullet lists under here are out of date

File: docs/guides/cloud/github-integration.mdx [hunk 4]
```diff
@@ -375,6 +375,17 @@ integration settings:
   alt="Status checks settings"
 />
 
+:::tip
+
+<strong>
+  <Icon name="star" /> Premium Cypress Cloud Feature
+</strong>
+
+Reporting on flakey status checks is included in our
```
Comment (pstakoun) [inline docs/guides/cloud/github-integration.mdx:384]:
```suggestion
Reporting on flaky status checks is included in our
```

File: docs/guides/cloud/gitlab-integration.mdx [hunk 5]
```diff
@@ -75,6 +102,22 @@ from merging any MRs with flaky tests.
...
+
+<strong>
+  <Icon name="star" /> Premium Cypress Cloud Feature
+</strong>
+
+Reporting on flakey status checks is included in our
```
Comment (pstakoun) [inline docs/guides/cloud/gitlab-integration.mdx:116]:
```suggestion
Reporting on flaky status checks is included in our
```

**Review by pstakoun — CHANGES_REQUESTED** [review]

File: docs/guides/cloud/github-integration.mdx (hunk 3 above)
Comment (elylucas) [reply docs/guides/cloud/github-integration.mdx:389]: Took a stab at updating the bullets to what I could gather. Let me know if you want something more specific.

**Review by pstakoun — APPROVED** [review]

**Pull request merged**